### DISCUSSION TITLE
AIR 50.0.0.1 cannot be used in Animate？

### DISCUSSION THREAD

**Discussion by Jason0247** [airsdk/Adobe-Runtime-Support#2189] [Q&A]
The press conference reported an error. The XML is still the AIR of version 33. Do you need to wait for the next update of version 50. X to use it? Or do I ignore the settings in which respect?

**Comment by itlancer** [comment]
@Jason0247, you can use workaround described here https://github.com/airsdk/Adobe-Runtime-Support/issues/2147
